packages/next/src/client/components/react-dev-overlay/app/ReactDevOverlay.tsx

```diff
@@ -27,15 +28,17 @@ export default class ReactDevOverlay extends React.PureComponent<
 > {
   state = { reactError: null }
 
-  static getDerivedStateFromError(error: Error): ReactDevOverlayState {
-    if (!error.stack) return { reactError: null }
+  static getDerivedStateFromError(err: Error): ReactDevOverlayState {
```

Comment:
Let's rename these to something meaningful. `err` vs `error` doesn't tell me what the difference is. Even worse, the shorter variable is the one you don't want to use mostly.